packages/web-react/src/components/Alert/stories/argTypes.ts

```diff
@@ -0,0 +1,23 @@
+import { EmotionColors } from '../../../constants';
+
+export default {
+  color: {
+    control: {
+      type: 'select',
+      options: [...Object.values(EmotionColors)],
+    },
+    defaultValue: EmotionColors.SUCCESS,
+  },
+  isCentered: {
+    control: {
+      type: 'boolean',
+    },
+    defaultValue: false,
+  },
+  iconName: {
+    control: {
+      type: 'text',
```

Comment:
Just a topic for discussion: What about adding a custom type with the icon select box? We know the finite set of icons, so we can provide an array of names.

The same can be done with the colors.

Reply:
Colors already come from select. So I added a select for three types of icons

Reply:
After a discussion with Michal Klimt, we agreed that all our icons could be here

Reply:
`iconName` shouldn't be available to change at all since that icon is color dependent, so it was removed

Reply:
👍